_includes/footer.html

```diff
@@ -39,8 +39,4 @@
   function off() {
     document.getElementById("overlay").style.display = "none";
   }
```

Comment:
i had this function below in so that it would show the first tab on load. Does the framework do that without this function now?

Reply:
Yeah bootstrap comes the dom looking for `.active`